.csscomb.json

```diff
@@ -1,24 +1,319 @@
 {
-    "remove-empty-rulesets": true,
     "always-semicolon": true,
-    "color-case": "lower",
     "block-indent": "    ",
+    "color-case": "lower",
     "color-shorthand": true,
     "element-case": "lower",
     "eof-newline": true,
-    "leading-zero": false,
+    "leading-zero": true,
     "quotes": "double",
+    "tab-size": true,
+    "remove-empty-rulesets": true,
+    "strip-spaces": true,
+    "unitless-zero": true,
+    "vendor-prefix-align": true,
+    "space-between-declarations": "\n",
     "space-before-colon": "",
     "space-after-colon": " ",
     "space-before-combinator": " ",
     "space-after-combinator": " ",
-    "space-between-declarations": " ",
     "space-before-opening-brace": " ",
     "space-after-opening-brace": "\n",
-    "space-after-selector-delimiter": " ",
     "space-before-selector-delimiter": "",
+    "space-after-selector-delimiter": "\n",
     "space-before-closing-brace": "\n",
-    "strip-spaces": true,
-    "tab-size": true,
-    "unitless-zero": true
+    "sort-order": [
+        "$variable",
```

Comment:
not sure about this thing however
it makes sense that local variables are defined on top of the declarations, but it can potentially break mixins that rely on recursive logic etc